invenio_records_resources/services/errors.py

```diff
@@ -20,6 +20,12 @@ class PermissionDeniedError(PermissionDenied):
 
     description = "Permission denied."
 
+    def __init__(self, action_name=None, record=None, *args, **kwargs):
+        """Initialize exception."""
+        self.record = record
+        self.action_name = action_name
+        super(PermissionDeniedError, self).__init__(*args, **kwargs)
```

Comment:
connected with the previous comment: is this parent constructor call matching parent class constructor signature? should it accept args and kwargs?